src/components/cl/basic/cl_basic_lib.c

```diff
@@ -66,5 +68,18 @@ ucc_status_t ucc_cl_basic_get_lib_attr(const ucc_base_lib_t *lib,
         attr->super.attr.coll_types  |= nccl_tl_attr.super.attr.coll_types;
     }
 
+    dpu_iface = ucc_get_component(&ucc_global_config.tl_framework, "dpu");
```

Comment:
Ok, this also must be generalized for arbitrary TLs case. I will make it in master and then this change will not be required.

Reply:
I guess I'll just wait until we remove it properly